Source/Phone/Phone.swift

```diff
@@ -931,7 +931,7 @@ public class Phone {
     }
 
     private func doLocusEvent(_ model: LocusModel) {
-        SDKLogger.shared.debug("Receive locus event: \(model.toJSONString(prettyPrint: self.debug) ?? nilJsonStr)")
+        //SDKLogger.shared.debug("Receive locus event: \(model.toJSONString(prettyPrint: self.debug) ?? nilJsonStr)")
```

Comment:
Should we just remove this instead of commenting out

Reply:
Sure, thanks.